src/commands/deploy.ts

```diff
@@ -371,91 +287,56 @@ To file a ticket, please head to: https://console.liara.ir/tickets`)
           this.log(`${attempt}: Could not cancel, retrying...`)
         }
       }
-      await retry(async () => {
-        await axios.post(`/v2/releases/${releaseID}/cancel`, null, this.axiosConfig)
-      }, retryOptions)
-
+      await cancelDeployment(this.got, releaseID, retryOptions)
       this.spinner.warn('Build canceled.')
       process.exit(3)
     })
 
-    return new Promise((resolve, reject) => {
-      const poller = new Poller()
-
-      let since: string
-      let isDeploying = false
-
-      poller.onPoll(async () => {
-        try {
-          const {data: {release, buildOutput}} = await axios.get<IBuildLogsResponse>(
-            `/v2/releases/${releaseID}/build-logs`, {
-              ...this.axiosConfig,
-              params: {since},
-            })
-
-          for (const output of buildOutput) {
-            this.spinner.clear().frame()
-
-            if (output.stream === 'STDOUT') {
-              process.stdout.write(output.line)
-            } else {
-              // tslint:disable-next-line: no-console
-              console.error(output.line)
-              return reject(new Error('Build failed.'))
-            }
-          }
-
-          if (!buildOutput.length) {
-            if (release.state === 'CANCELED') {
-              this.spinner.warn('Build canceled.')
-              process.exit(3)
-            }
-
-            if (release.state === 'TIMEDOUT') {
-              this.spinner.fail();
-              return reject(new Error('TIMEOUT'))
-            }
-
-            if (release.state === 'FAILED') {
-              this.spinner.fail();
-              if(release.failReason) {
-                return reject(new DeployException(this.parseFailReason(release.failReason)))
-              }
-              return reject(new Error('Release failed.'))
```

Comment:
Where's this logic? You have removed it altogether.

Reply:
I've tried to revert the exception where I left off